pennylane_qiskit/qiskit_device2.py

```diff
@@ -266,7 +263,7 @@ def __init__(
             raise ValueError(f"Backend '{backend}' supports maximum {available_qubits} wires")
 
         self.reset()
-        # ToDo: process or update kwargs in some fashion here
+        self._process_kwargs()  # processes kwargs and separates transpilation arguments to dev._transpile_args
```

Comment:
I would prefer this to return the `kwargs` and `transpiler_kwargs` instead of them changing it outside the `__init__` context. 
```suggestion
       self._transpile_args, self._kwargs = self._process_kwargs(kwargs)  # processes kwargs and separates transpilation arguments to dev._transpile_args
```

Reply:
Makes sense. I think that's cleaner as well.